bin/ncp/NETWORKING/nc-static-IP.sh

```diff
@@ -86,8 +86,9 @@ EOF
     systemctl restart networking
   }
  
-  sudo -u www-data php /var/www/nextcloud/occ config:system:set trusted_domains 1 --value="$IP"
-  sudo -u www-data php /var/www/nextcloud/occ config:system:set overwrite.cli.url --value=https://"$IP"/
+  ncc config:system:set trusted_domains "${TRUSTED_DOMAINS[ip]}" --value="$IP"
+  # TODO: Should we really overwrite potentially configured domains here?
+  set-nc-domain "$IP"
```

Comment:
Do we really want to overwrite the nc domain with the static IP? Especially since the IP is only reachable locally, that could be problematic. I'd suggest to only add it to trusted_domains but not set overwrite.cli.url

Reply:
I think it's that way so we can use pretty urls in this case